procedural/examples/foo-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/fooprocedures/procedures/SampleProcedure.java

```diff
@@ -19,9 +21,12 @@ public void run(@NotNull final EditablePlan plan) {
     var currentTime = firstTime;
     for (var i = 0; i < quantity; i++) {
       plan.create(
-          "BiteBanana",
-          new DirectiveStart.Absolute(currentTime),
-          Map.of()
+          new NewDirective(
+              new AnyDirective(Map.of()),
+              "It's a bite banana activity",
```

Comment:
Make sure to add this name field insertion to your new procedures in your e2e test improvement branch 👍